ts-client/src/dlmm/index.ts

```diff
@@ -1084,6 +1101,12 @@ export class DLMM {
     );
     const program = new Program(IDL, LBCLMM_PROGRAM_IDS[opt.cluster], provider);
 
+    if (
+      !this.checkPoolExists(connection, tokenX, tokenY, binStep, baseFactor)
```

Comment:
could you renaming function to: isPoolNotExisted ?

Reply:
renamed to `isPoolExistsed` 👍